pkg/util/workqueue/prometheus/prometheus.go

```diff
@@ -85,3 +85,25 @@ func (prometheusMetricsProvider) NewRetriesMetric(name string) workqueue.Counter
 	prometheus.Register(retries)
 	return retries
 }
+
+func (prometheusMetricsProvider) NewLongestRunningProcessorSecondsMetric(name string) workqueue.SettableGaugeMetric {
```

Comment:
Please update the entire file with https://github.com/kubernetes/kubernetes/blob/v1.16.0/pkg/util/workqueue/prometheus/prometheus.go. The prometheus metrics conventions has been changed since 1.14 and the subsystem configured here  doesn't match to new convention since 1.16. 
Ref.: https://github.com/kubernetes/kubernetes/pull/77553

Reply:
Done refer - https://github.com/gardener/machine-controller-manager/pull/398/commits/f710c440a050c54fe14dae3ba7d00009b14ffb2c.